.changeset/ripe-cooks-yawn.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fluid-tools/fetch-tool": minor
+"@fluidframework/odsp-urlresolver": minor
+---
```

Comment:
I _think_ the suggestion below is valid... can't recall if it might cause issues during the release process (maybe if we try to generate changelogs in the individual affected packages, since this one will be gone; cc @tylerbutler @dannimad @jatgarg @CraigMacomber to see if one of you remembers how we should announce packages going away).

In any case, I don't think we need to call out fetch-tool or odsp-urlresolver as affected.

```suggestion
---
"@fluid-tools/fluidapp-odsp-urlresolver": major
---
```

Reply:
Well the `pnpm changeset` doesn't provide @fluid-tools/fluidapp-odsp-urlresolver as option when choosing package. Although I think it does make sense to have announcement in your example way.